core/src/test/scala/unit/kafka/admin/AddPartitionsTest.scala

```diff
@@ -185,9 +184,8 @@ class AddPartitionsTest extends BaseRequestTest {
     assertTrue(s"Partition $partitionId should exist", partitionOpt.isDefined)
     val partition = partitionOpt.get
 
-    assertNotNull("Partition leader should exist", partition.leader)
```

Comment:
Why did we remove this? Thanks for simplifying.

Reply:
There is no longer a `leader` field. Below we verify the leaderId directly.